project-base/storefront/components/Blocks/Product/ProductsSlider.tsx

```diff
@@ -74,6 +81,10 @@ export const ProductsSlider: FC<ProductsSliderProps> = ({
                         gtmProductListName={gtmProductListName}
                         gtmMessageOrigin={gtmMessageOrigin}
                         listIndex={index}
+                        isProductInComparison={isProductInComparison(productItemData.uuid)}
+                        onProductInComparisonClick={() => handleProductInComparison(productItemData.uuid)}
+                        handleProductInWishlist={() => handleProductInWishlist(productItemData.uuid)}
+                        isInWishlist={isProductInWishlist(productItemData.uuid)}
```

Comment:
My OCD cried a little here. Could we somehow unify the names of these props? Basically they are 2x the same thing, but the names are very different.

Reply:
I hope your CDO (in alphabetical order, as all things should be) is now a little more satisfied – 8fdca51562

(if you think it should be named differently, I'm open to suggestions)

Reply:
Great!